### PR TITLE
sci-calculators/bc-gh: bump to 4.0.0

### PR DESCRIPTION
**Pull request by strikersh**
Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Jules de Sartiges <desartij@protonmail.com>

### CODE REVIEW

File: sci-calculators/bc-gh/bc-gh-4.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Gentoo Authors
```
Comment (scardracs) [inline sci-calculators/bc-gh/bc-gh-4.0.0.ebuild:1]: Strange, repoman should report this to you. It is 1999-2021

Comment (ionenwks) [reply sci-calculators/bc-gh/bc-gh-4.0.0.ebuild:1]: > Strange, repoman should report this to you. It is 1999-2021

repoman/pkgcheck currently only care if date is outdated, but [devmanual](https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#ebuild-header) wants the header.txt as-is

Comment (strikersh) [reply sci-calculators/bc-gh/bc-gh-4.0.0.ebuild]: I did update the new ebuild should I update the others?

**Review by juippis — APPROVED** [review]

File: sci-calculators/bc-gh/bc-gh-4.0.0.ebuild (hunk 1 above)
Comment (juippis) [reply sci-calculators/bc-gh/bc-gh-4.0.0.ebuild]: Does your own repoman complain about it? Please test before making unsure review comments.

**Comment by strikersh** [comment]
why was it closed?

**Comment by juippis** [comment]
gentoo-bot gave a link why it was closed.

We don't use this Github repo, it's only a **mirror**. So we apply these .patch files into our own repo, and close the PRs with commits. We don't merge to Github repo.

**Pull request closed without merging**